### PR TITLE
APIS-6671

### PR DESCRIPTION
**Pull request by peteslater-ee**

### CODE REVIEW

**Review by mattclark-zerogravit — DISMISSED** [review]
👍

File: project/plugins.sbt [hunk 1]
```diff
@@ -3,8 +3,8 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
 
 resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
 
-addSbtPlugin("uk.gov.hmrc"      % "sbt-auto-build"              % "3.8.0")
+addSbtPlugin("uk.gov.hmrc"      % "sbt-auto-build"              % "3.18.0")
 addSbtPlugin("uk.gov.hmrc"      % "sbt-play-cross-compilation"  % "2.3.0")
```
Comment (mattclark-zerogravit) [inline project/plugins.sbt]: Does this need removing?

Comment (peteslater-ee) [reply project/plugins.sbt]: 👍

**Review by mattclark-zerogravit — APPROVED** [review]
👍

**Pull request merged**